lang/test/org/partiql/lang/util/testdsl/IonResultTestCase.kt

```diff
@@ -71,9 +72,16 @@ data class IonResultTestCase(
 internal fun IonResultTestCase.runTestCase(
     valueFactory: ExprValueFactory,
     db: MockDb,
+    target: EvaluatorTestTarget,
     compilerPipelineBuilderBlock: CompilerPipeline.Builder.() -> Unit = { }
 ) {
-    val harness: EvaluatorTestAdapter = AstEvaluatorTestAdapter()
+    val adapter = PipelineEvaluatorTestAdapter(
+        when (target) {
+            EvaluatorTestTarget.COMPILER_PIPELINE -> CompilerPipelineFactory()
+            EvaluatorTestTarget.PLANNER_PIPELINE -> TODO("PlannerPipelineFactory()")
+            EvaluatorTestTarget.ALL_PIPELINES -> error("May only test one pipeline at a time with IonResultTestCase")
```

Comment:
So why can only one pipeline test at a time w/ `IonResultTestCase`? Will that be added in a subsequent PR?

Reply:
There was a reason, I'll have to look remind myself of what it is and add a comment here to explain it.

Reply:
The reason has to do with each pipeline needing a different skip list which is accounted for elsewhere before this function is invoked.  I'll add a comment.